addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.xml

```diff
@@ -48,7 +48,7 @@
                 <t t-foreach="slot.record.fieldNames" t-as="fieldName" t-key="fieldName">
                     <t t-if="!slot.record.isInvisible(fieldName)">
                         <t t-set="fieldInfo" t-value="props.model.popoverFields[fieldName]" />
-                        <li class="list-group-item flex-shrink-0 d-flex flex-wrap align-items-center" t-att-class="fieldInfo.rawAttrs.class">
+                        <li class="list-group-item d-flex text-nowrap align-items-center" t-att-class="fieldInfo.rawAttrs.class" t-att-data-tooltip="props.record.rawRecord[fieldName][1]">
```

Comment:
`t-att-data-tooltip="props.record.rawRecord[fieldName][1]"` assumes that the field is a m2o which is not always the case in other app than planning. You need to use the formatters.
```js
const fieldType = fieldInfo.widget || fieldInfo.field.type;
const format = registry.category("formatters").get(fieldType);
const formattedValue = format(props.record.rawRecord[fieldName]);
```

Reply:
Ah true! 🙈